### ISSUE TITLE
interleave between voices - rests not recognized as common(?)

### ISSUE THREAD

**Issue by Bacchushlg** [Audiveris/audiveris#307]
[Run to You.zip](https://github.com/Audiveris/audiveris/files/3693047/Run.to.You.zip)

The appended score makes problems concerning common rests for 2 voices. When imported to MuseScore there is an interleaved starting from measure 18 in the bottom 2 note lines: while one voice is correct, the other one has it's note of tick 4 always on tick 3, as if the common rest had not been recognized as such.
Inside Audiveris everything was fine.
I'm afraid that this effect is new - I think I had similar score in the past that did not have this problem.
Version tested: 5.2.0-alpha:3d572694e

**Comment by hbitteur** [comment]
I have a problem with your input file, sheet number 2 has no BINARY.xml component, while all the other sheets have their binary counterpart. 
I cannot display sheet 2 where I think measure 18 is located.

Could you reprocess this score and check that the .omr file contains the required components.
Sorry Baruch for the additional work ... :-(

**Comment by Bacchushlg** [comment]
I assume the reason is that I had reset the sheet to binary and restarted recognition...
But the problem reappears on the following pages. Maybe you see there?

Am 6. Oktober 2019 22:14:33 MESZ schrieb "Hervé Bitteur" <notifications@github.com>:

**Comment by hbitteur** [comment]
Could you send me the original input file? 
Because, without a sheet binary, I have no means to reprocess or even load the sheet.

I will have to investigate why such binary.xml component is not present. Perhaps because of a binary reset as you said. This is another problem to work upon. But let's address your initial problem first.

**Comment by Bacchushlg** [comment]
I'm not at home for the rest of the week. I send the file when I'm back.

Am 7. Oktober 2019 09:46:57 MESZ schrieb "Hervé Bitteur" <notifications@github.com>:

**Comment by Bacchushlg** [comment]
[Run to You.zip](https://github.com/Audiveris/audiveris/files/3716510/Run.to.You.zip)

**Comment by hbitteur** [comment]
Yes, there is yet no notion of rest common to several voices in Audiveris.
See #380
